src/model/parseModelDefinition.ts

```diff
@@ -58,6 +60,14 @@ export function parseModelDefinition<Dictionary extends ModelDictionary>(
         return result
       }
 
+      if (isStrictlyObject(definition[property])) {
```

Comment:
will check

Reply:
In case it doesn't, please don't hurry to implement array support. I like the work done in #98, and I'd love to see both of your contributions gradually combined.

Reply:
it seem to work, for my expectations at least

Reply:
Thanks for checking!